javascript/javascript1/week1/Homework.js

```diff
@@ -1,5 +1,5 @@
 console.log("Age-ify");
 let yearOfBirth = 1997;
 let yearFuture = 2121;
-const age = yearFuture - yearOfBirth;
+let age = yearFuture - yearOfBirth;
 console.log("You will be " + age + " years old in " + yearFuture);
```

Comment:
Good to see you updated the variable age from const to let. 👌

Is there another variable in this code that could be a const? 😊 
- Hint: a number that would never change even as the years are passing by.

Reply:
I think yearOfBirth should be const looking back on this - as it shouldn't change, I'll go back and amend that :D